proposed/http-message-meta.md

```diff
@@ -163,7 +163,7 @@ messages.
 
 ## 5. Design Decisions
 
-### Message design
+### 5.1 Message design
```

Comment:
Shouldn't we ending headline numbers with `.` just like `5. Design Decisions` did?